study.md

```diff
@@ -56,13 +60,13 @@ If a class in Ruby gets modified while there are existing instances of it, then
 will those instances also have those modifications?
 
 ```md
-<!-- your answer here -->
+No. Those instances can only be modified within that instance.
 ```
 
 ## Iteration
 
 Write an example of iteration in Ruby.
 
 ```ruby
-# your answer here
-```
+variable x = 1.upto(10) { |i| + i*5}
```

Comment:
This will cause a syntax error due to `variable` being undefined and the body of the block `+ i*5` being invalid.